errors/multierror.go

```diff
@@ -32,5 +32,10 @@ func (errs MultiError) Error() string {
 		}
 	}
 
-	return fmt.Sprintf("Hit multiple errors:\n%s", strings.Join(errorMessages, "\n"))
+	for _, errStatus := range errorMessages {
+        if errStatus == "exit status 2" {
+			return errStatus
+		}
```

Comment:
Not sure I follow the logic here. Why in `MultiError` would we only return this error message if it was present? And does this actually change how Terragrunt handles this exit code?

Reply:
Let me do a more thorough investigation on how this exit code is handled and see how best to fix this issue.